docs/index.rst

```diff
@@ -44,6 +44,11 @@
 .. raw:: html
    :file: html/hero.html
 
+--------
+Latest news
+--------
+
+.. include:: _rst_includes/latest_news.rst
```

Comment:
perhaps the latest_news.rst file should also contain the title, in this way you can check if there is any news and adapt the full block accordingly, hmm...

Reply:
The latest news title, i.e. The entire latest news section